package.json

```diff
@@ -4,7 +4,8 @@
   "description": "A lightweight alternative for consuming GraphQL. Zero caching, zero bells. Zero Bloat.",
   "main": "index.js",
   "scripts": {
-    "test": "jest"
+    "test": "jest",
+    "build": "npx pack build"
```

Comment:
I think this needs to be `pack build` not `npx pack build` (this will download and execute the `pack` npm lib, which is different from `@pika/pack`.

Reply:
But when you install `@pika/pack` locally, the bin it installs is `node_modules/.bin/pack`, so omitting “npx” should be right, I think

Reply:
Interesting. It works :man_shrugging: 

If I do it your way presumably I need to install `@pika/pack` as a dev dependency. Which is fine, I just currently have it installed globally as shown in their introductory blog post.

Reply:
Oh! Yeah I didn’t think about that. Yeah when I don’t have it installed globally, I get:

```
npx pack
npx: installed 1 in 1.609s
command not found: pack
```